qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -429,3 +412,85 @@ def operated_qubits(self) -> Dict[int, List[int]]:
                 self._operated_qubits[len(coupling)].append(coupling)
 
         return self._operated_qubits
+
+    def serialize(self, save_parameters: bool = True) -> Dict:
+        """Serializes the class to a Dictionary.
+
+        Args:
+            save_parameters: If set to True, the default value, then all the values of the
+                calibrations will also be serialized.
+
+        Returns:
+            A dict object that represents the calibrations and can be used to rebuild the
+            calibrations. See :meth:`deserialize`.
+
+        Raises:
+            CalibrationError: if there calibrations were not built from a library.
+        """
+
+        if self._library is None:
+            raise CalibrationError(
+                "Cannot serialize calibrations that are not constructed from a library."
+            )
+
+        serialized_cals = serialize_object(type(self))
+        serialized_cals["__value__"]["__library__"] = self._library.serialize()
+        serialized_cals["__value__"]["__backend_name__"] = self._backend.name()
+        serialized_cals["__value__"]["__backend_version__"] = self._backend.version
+
+        if save_parameters:
+            serialized_cals["__value__"]["__parameter_values__"] = self.parameters_table()["data"]
+
+        return serialized_cals
+
+    # pylint: disable=arguments-differ
+    @classmethod
+    def deserialize(cls, serialized_dict: Dict, backend: Backend, *args) -> "BackendCalibrations":
+        """Deserialize from a dictionary.
+
+        Args:
+            serialized_dict: The dictionary from which to create the calibrations instance.
+            backend: The backend instance from which to construct the calibrations.
+            args: Trailing args.
```

Comment:
When is `args` used?

Reply:
I removed it. I also removed the serialization of the `Calibrations` class. Now, only `BackendCalibrations` can be serialized. See 72ecf98 and 492ae19.